Documentation/botr/readytorun-format.md

```diff
@@ -79,9 +77,9 @@ native code from image of version 3.0.
 
 ### READYTORUN_HEADER::Flags
 
-| Flag | Value | Description |
-|:-----|------:|:------------|
-| ``READYTORUN_FLAG_PLATFORM_NEUTRAL_SOURCE`` | 0x00000001 | Set if the original IL image was platform neutral. The platform neutrality is part of assembly name. This flag can be used to reconstruct the full original assembly name. |
+| Flag                                    |      Value | Description                                                                                                                                                                |
+|:----------------------------------------|-----------:|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
Hmm, these long lines do not make it much readable as a raw text in simple editor either. I guess it may better to leave the tables with long multi-line cells without any indentation. I am sorry for the back-and-forth.

Reply:
No worries, I myself am learning the markdown syntax as I go. One compromise option that occurs to me is as follows: I found out in the documentation that the outer vertical bars are optional. So I'd just leave them out and I'd shorten the minuses to just cover the header row. Apart from being visually somewhat closer to the rich-formatted md tables, I believe it would make the tables more readable in text mode too. I'll try that out in 8th commit and if you don't like it, I'll revert it back to the original version.

Reply:
OK, slight correction. I had to keep the initial vertical bar as otherwise the parser doesn't recognize the table if the leftmost column is right-aligned (typically for numeric values) and I believe that artificially left-aligning them just to satisfy the parser is not visually pleasing in the text view. For now I'm relatively content that the updated version in 8th commit looks much better than all the previous versions.

Reply:
Agree. It looks pretty good now.